src/components/WithdrawPage.js

```diff
@@ -256,6 +256,8 @@ const WithdrawPage = (props: { dappGateway: DappGateway, noWeb3: Boolean, noCont
             })
             .filter(x => x[0].cmp(bnZero) !== 0 && x[1].cmp(bnZero) !== 0)
 
+          console.log(publicKeys)
```

Comment:
Is it good to log stuff out in the production app?

Reply:
Nice catch ahahaha, its was me being sloppy